### PR TITLE
[FEATURE 💡] Remove Nike from header in mobile devices

### LINKED ISSUE #67: [FEATURE 💡] Remove Nike from header in mobile devices
### Description

- Remove the nike text from the logo **(Only for mobile devices)**
- Make the navbar a bit shorter for **(Only for mobile devices)**
- The logo and icons must also be made smaller 
- The theme icon seems not aligned properly
- Take reference from the original site webpage screenshots given below

### Screenshots

image

- Reference
image

## Happy contributing. 💐
Star my other Repositories [here](https://github.com/warmachine028)

### PR DESCRIPTION
**Pull request by mehulrnsit**
# Issue #67  📐

- I [Mehul](https://github.com/mehulrnsit) have worked for #67 

## Guidelines 🔐

I accept the fact that I have followed the guidelines and have not copied the codes from around the internet

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- My pull request closes #67 

## Screenshots 📷

Here are the pictures of changes that I have made 🔽

image

### CODE REVIEW

File: src/components/Navbar.jsx
Comment (warmachine028) [inline src/components/Navbar.jsx:1]: @mehulrnsit provide screenshots for desktop devices also, to verify that the changes are only made for mobile devices.

**Review by warmachine028 — CHANGES_REQUESTED** [review]
@mehulrnsit provide screenshots for desktop devices also, to verify that the changes are only made for mobile devices.

File: src/components/Navbar.jsx
Comment (mehulrnsit) [reply src/components/Navbar.jsx:1]: Attaching Desktop screenshot below
image

Comment (warmachine028) [reply src/components/Navbar.jsx:1]: Looks Good

**Review by warmachine028 — APPROVED** [review]
All changes look good

**Comment by warmachine028** [comment]
## Fastastic work @mehulrnsit! I'll be merging it right away.
- Make sure to star this repo 🌟 if you liked contributing to it.
- You can also follow me for more such amazing projects and opportunities.
- I have added the hacktoberfest-accepted tag also

**Comment by mehulrnsit** [comment]
Thanks :)

**Pull request merged**